src/lib/parquet/tests/Flow/Parquet/Tests/Integration/IO/CompressionTest.php

```diff
@@ -15,9 +16,11 @@
 
 final class CompressionTest extends TestCase
 {
+    use FilesystemTestHelper;
```

Comment:
There is a small problem here, libraries should not include anything from ETL itself, I'm planning to implement architecture constraints, and this one will fail. Alternatively we can create another lib and just expose FilesystemTestHelper there, it would also be a good place for PHPUnit helpers that I mentioned some time ago, plus maybe some more goodies that would help us when creating tests, like for example generating rows that include all possible column types in one place.